src/resolve/resolved-dns-stub.c

```diff
@@ -949,6 +949,12 @@ static void dns_stub_process_query(Manager *m, DnsStubListenerExtra *l, DnsStrea
                 return;
         }
 
+        if (set_contains(m->refuse_record_types, INT_TO_PTR(dns_question_first_key(p->question)->type))) {
+                log_debug("Got request that is refused, refusing.");
```

Comment:
please include an indication which rr type we are talking about here, i.e. `dns_type_to_string(dns_question_first_key(p->question)->type)`